HowTos/create_vpc.rst

```diff
@@ -1,39 +1,58 @@
- 
 Aviatrix Transit VPC
 ----------------------
 
-When the "Aviatrix Transit VPC" option is selected, Create a VPC creates the following AWS VPC subnets.
+When the "Aviatrix Transit VPC" option is selected, all necessary subnets and route tables will be fully populated as below:
 
 The VPC CIDR range for a Transit VPC is from /16 to /23.
 
 ==========================================      ===================
 **Subnet name**                                 **Suggested usage**
 ==========================================      ===================
-Public-gateway-and-firewall-mgmt-az1            Use this subnet to launch Aviatrix primary transit gateway.Use this subnet to launch firewall instance in DMZ deployment. 
-Public-gateway-and-firewall-mgmt-az2            Use this subnet to launch backup Aviatrix transit gateway. Use this subnet to launch backup firewall instance in a second availability zone in DMZ deployment.
+Public-gateway-and-firewall-mgmt-az1            Use this subnet to launch Aviatrix primary transit gateway. Use this subnet to launch firewall instance in DMZ deployment. 
+Public-gateway-and-firewall-mgmt-az2            Use this subnet to launch Aviatrix backup transit gateway. Use this subnet to launch backup firewall instance in a second availability zone in DMZ deployment.
 Private-FW-north-az1                            Use this subnet to create an interface on primary firewall instance that interacts with Aviatrix Main gateway in DMZ deployment.
 Private-FW-north-az2                            Use this subnet to create an interface on backup firewall instance that interacts with Aviatrix Main gateway in DMZ deployment.
 Private-FW-south-az1                            Use this subnet to create an interface on primary firewall instance that interacts with Aviatrix Companion gateway in DMZ deployment.
 Private-FW-south-az2                            Use this subnet to create an interface on backup firewall instance that interacts with Aviatrix Companion gateway in DMZ deployment.
-Private-FW-ingress-egress-az1                   Use this subnet to create an interface on primary firewall instance handles ingress and egress traffic in DMZ deployment.
-Private-FW-ingress-egress-az2                   Use this subnet to create an interface on backup firewall instance handles ingress and egress traffic in DMZ deployment.
+Public-FW-ingress-egress-az1                    Use this subnet to create an interface on primary firewall instance handles ingress and egress traffic in DMZ deployment.
+Public-FW-ingress-egress-az2                    Use this subnet to create an interface on backup firewall instance handles ingress and egress traffic in DMZ deployment.
+==========================================      ===================
+
+Aviatrix FireNet VPC/VNet
+-------------------------
+
+When the "Aviatrix FireNet VPC" or "Aviatrix FireNet VNet" option is selected, all necessary subnets and route tables will be fully populated as below:
```

Comment:
Suggest a change to the word "below" to "follows".

..will be fully populated as follows: